stepactions/bundles/pick-cluster-version/0.1/README.MD

```diff
@@ -0,0 +1,15 @@
+# pick-cluster-version stepaction
```

Comment:
I find the Example Usage section of some of these other step-actions to be very useful. Could you add that to this readme

For example something like this https://github.com/konflux-ci/tekton-integration-catalog/tree/main/stepactions/sealights/upload-ginkgo-results/0.1#example-usage